### PR TITLE
Kwan Hao Wei - Update About Us and PPP Page

### PR DESCRIPTION
**Pull request by KwanHW**
Fixes
- #53 
- #57

### CODE REVIEW

File: docs/team/kwanhw.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+### Project: ContaX
+
+ContacX is for professionals who have many clients, current and prospective, who may possibly work with sensitive personal information and documents related to each of those clients. They have busy schedules with many appointments & meetings, and must be able to find available time slots in their schedules quickly.
+
+ContacX will allow for efficient management of a large list of contacts, allowing the mental workload of remembering people and related scheduled tasks to be offloaded. It will also provide an efficient way of efficiently organizing resources, while having minimal downtime with seamless transition from existing solutions.
+
+Given below are my contributions to the project.
+
+-   **New Feature**: Onboarding guide for new users
```
Comment (sharpstorm) [inline docs/team/kwanhw.md:14]: Are you sure about this?

File: docs/AboutUs.md [hunk 2]
```diff
@@ -35,18 +35,18 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
+<img src="images/kwanhw.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/kwanhw)]
+[[portfolio](team/kwanhw.md)]
 
-* Role: Developer
-* Responsibilities: Dev Ops + Threading
+-   Role: Developer
```
Comment (sharpstorm) [inline docs/AboutUs.md:48]: Please change to asterisk (for standardization across the file).

**Review by sharpstorm — CHANGES_REQUESTED** [review]
PPP skeleton includes features that do not look right, please check.

File: docs/team/kwanhw.md (hunk 1 above)
Comment (KwanHW) [reply docs/team/kwanhw.md]: Fixed in 33e64e3

File: docs/AboutUs.md (hunk 2 above)
Comment (KwanHW) [reply docs/AboutUs.md]: Fixed in cb3b9c5

**Review by sharpstorm — APPROVED** [review]
LGTM.

**Review by dandaandaaaaaan — APPROVED** [review]
lgtm!

**Pull request merged**